extensions/sql-database-projects/src/test/baselines/openSdkStyleSqlProjectWithFilesSpecifiedBaseline.xml

```diff
@@ -35,7 +35,7 @@
   </ItemGroup>
   <ItemGroup>
     <PostDeploy Include="Script.PostDeployment1.sql" />
-    <None Include="folder1\Script.PostDeployment1.sql" />
+    <None Include="folder1\Script.PostDeployment2.sql" />
```

Comment:
Why?  Shouldn't this still work if the file has the same name but is in a different location?